storage/innobase/xtrabackup/test/t/xb_export.sh

```diff
@@ -126,23 +131,35 @@ start_server $mysql_extra_args
 init_schema incremental_sample test $max_indices_count "n"
 init_schema incremental_sample test2 $indices_count "n"
 init_schema incremental_sample test3 $indices_count "y"
+# check if we can import all the columns type in mysql
+mysql -e 'create table test4(col1 bool, col2 int, col3 float, col4 double,
+col5 timestamp, col6 long, col7 date, col8 time, col9 datetime, col10 year,
+col11 varchar(20), col12 bit ,col13 decimal, col14 blob, col16 json,
+col17 mediumtext, col18 enum("01","2"),col19 SET("0","1","2") )' incremental_sample;
 vlog "Database was re-initialized"
 
 mysql -e "alter table test discard tablespace;" incremental_sample
 mysql -e "alter table test2 discard tablespace;" incremental_sample
 mysql -e "alter table test3 discard tablespace;" incremental_sample
+mysql -e "alter table test4 discard tablespace;" incremental_sample
 
 xtrabackup --datadir=$mysql_datadir --prepare --export \
     --target-dir=$backup_dir
 
+ls -tlr $backup_dir/incremental_sample/*cfg
 cfg_count=`find $backup_dir/incremental_sample -name '*.cfg' | wc -l`
-vlog "Verifying .cfg files in backup, expecting 5."
-test "$cfg_count" -eq 5
+vlog "Verifying .cfg files in backup, expecting 3. currently "$cfg_count
+if [ "$cfg_count" -ne 3  ]
+then
+    vlog "cfg files are not equal to 3 "
+    exit -1
+fi
```

Comment:
why the count decreased from 5 to 3?

Reply:
fixed the test case. some confusion because of test.